### ISSUE TITLE
Train the neural renderer

### ISSUE THREAD

**Issue by RongkaiZhang** [MarkMoHR/virtual_sketching#3]
Hi Mark,

Thank you for your excellent and impressive work.
I would like to train the neural renderer myself. However, if I generate a random stroke (i.e. a single fixed-width line, say wideth=3) and use L-2 loss, the trained neural renderer will output a solid dark image. I think it is due to the small difference between the dark image and the one with a single line. I cannot find the code for training the neural renderer. Could you please share small ideas or the code you used to train it?

Best regards,
Rongkai

**Comment by MarkMoHR** [comment]
Hi,

I trained the neural renderer with the approach of [Learning-to-Paint](https://github.com/megvii-research/ICCV2019-LearningToPaint). You can try it.

Regarding your issue, I think you can also try L1 loss or Binary Cross Entropy loss. And I recommend fine-tuning on the provided model.
